build.gradle

```diff
@@ -320,7 +318,7 @@ task showGrpcJavaPluginPath << {
 task showProtobufPath() << {
   DependencyResolver resolver = new DependencyResolver(project)
   println 'output: ' + resolver.extractArchive(
-    'com.google.protobuf:protobuf-java:3.0.0-beta-2');
+    'com.google.protobuf:protobuf-java:3.0.0-beta-3');
```

Comment:
@garrettjonesgoogle I don't know that this makes much difference, but it seems good that this move in lockstep with `protobuf`. What do you think?